.editorconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 the original author or authors.
```

Comment:
This goes for all of the copyright headers, this should be changed to the proper copyright:
```
Copyright 2016 John Grosh (jagrosh).
```